pkgs/top-level/all-packages.nix

```diff
@@ -20013,10 +20013,12 @@ with pkgs;
 
   leafnode = callPackage ../servers/news/leafnode { };
 
-  lemmy = callPackage ../servers/web-apps/lemmy {
```

Comment:
Will be nice to create an alias in `pkgs/top-level/aliases.nix` for this change.

Reply:
the package was added a couple of weeks ago though, I think we can save an alias here.